c8/client.py

```diff
@@ -2749,6 +2749,12 @@ def remove_key_value_pairs(self, name):
 
     # client.create_api_key
 
+    def get_jwt(self, email, password, tenant=None, username=None):
+        payload = {"email": email, "password": password}
+        if tenant is not None and username is not None:
```

Comment:
It's not tenant "and"  username.. we can pass either tenant, or  username or both or none.. so I guess two separate if statement would be needed..
```suggestion
        if tenant is not None:
            payload["tenant"] = tenant
```